src/main/kotlin/io/github/yuutoproject/yuutobot/commands/Dialog.kt

```diff
@@ -20,45 +20,144 @@ package io.github.yuutoproject.yuutobot.commands
 
 import io.github.yuutoproject.yuutobot.commands.base.AbstractCommand
 import io.github.yuutoproject.yuutobot.commands.base.CommandCategory
+import io.github.yuutoproject.yuutobot.utils.EMOJI_REGEX
+import io.github.yuutoproject.yuutobot.utils.EMOTE_MENTIONS_REGEX
+import io.github.yuutoproject.yuutobot.utils.NONASCII_REGEX
+import io.github.yuutoproject.yuutobot.utils.httpClient
 import java.io.IOException
 import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent
 import net.dv8tion.jda.internal.utils.IOUtil
 import okhttp3.*
 import okhttp3.Callback as OkHttp3Callback
 import okhttp3.MediaType.Companion.toMediaType
 import okhttp3.RequestBody.Companion.toRequestBody
+import org.slf4j.LoggerFactory
 
-// TODO: get arch to implement the actual command
 class Dialog : AbstractCommand("dialog", CommandCategory.INFO, "does something", "[bg] <char> <text>") {
+    private val logger = LoggerFactory.getLogger(this.javaClass)
+
+    private val backgrounds: List<String> = listOf(
```

Comment:
No need to be explicit about it, kotlin is smart
```suggestion
    private val backgrounds = listOf(
```